.eslintrc

```diff
@@ -0,0 +1,332 @@
+{
+    "parser": "babel-eslint",
+    "plugins": [
+    ],
+    "env": {
+        "browser": true,
+        "node": true,
+        "jasmine": true,
+        "es6": true,
+        "jest": true
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "arrowFunctions": true,
+            "blockBindings": true,
+            "classes": true,
+            "defaultParams": true,
+            "destructuring": true,
+            "forOf": true,
+            "generators": false,
+            "modules": true,
+            "objectLiteralComputedProperties": true,
+            "objectLiteralDuplicateProperties": false,
+            "objectLiteralShorthandMethods": true,
+            "objectLiteralShorthandProperties": true,
+            "restParams": true,
+            "spread": true,
+            "superInFunctions": true,
+            "templateStrings": true,
+        }
+    },
+    "rules": {
```

Comment:
you can extend plugins like airbnb recommended, typescript recommended etc, not to decline all the rules yourself, just to override any of them